src/main/java/Task.java

```diff
@@ -0,0 +1,24 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected String identity;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        identity = "[ ]";
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[X]" : "[ ]");
+    }
+    public String getContent(){
```

Comment:
native Java toString() method could be used so that method does not need to be called when print

Reply:
OK